yt_dlp/extractor/pbs.py

```diff
@@ -199,25 +200,37 @@ class PBSIE(InfoExtractor):
 
     _TESTS = [
         {
-            'url': 'http://www.pbs.org/tpt/constitution-usa-peter-sagal/watch/a-more-perfect-union/',
-            'md5': '173dc391afd361fa72eab5d3d918968d',
+            'url': 'https://www.thirteen.org/programs/the-woodwrights-shop/carving-away-with-mary-may-tioglz/',
             'info_dict': {
-                'id': '2365006249',
+                'id': '3004803331',
                 'ext': 'mp4',
-                'title': 'Constitution USA with Peter Sagal - A More Perfect Union',
-                'description': 'md5:31b664af3c65fd07fa460d306b837d00',
-                'duration': 3190,
+                'title': "The Woodwright's Shop - Carving Away with Mary May",
+                'description': 'md5:7cbaaaa8b9bcc78bd8f0e31911644e28',
+                'duration': 1606,
+                'display_id': 'carving-away-with-mary-may-tioglz',
+                'chapters': [],
+                'thumbnail': 'https://image.pbs.org/video-assets/NcnTxNl-asset-mezzanine-16x9-K0Keoyv.jpg',
             },
         },
```

Comment:
add this as a new test, don't replace the existing test. if the existing test's link is dead, then add a `skip` field to the test dict

Reply:
Hmm. I did this, but `hatch test PBSIE` now seems to skip all tests in the dict, not only the single one…? (If I try to produce errors in the other tests, the test run still doesn't report errors, only "SKIPPED 1 test")

Reply:
`hatch test PBS` only runs the first test case. use `PBS_all` or `PBS_1`, `PBS_2` etc

and don't force-push please

Reply:
Well, I somehow had to amend my first commit, can't do that without force pushing.

Thanks for the `PBS_all` hint… unfortunately several more tests now fail, and I don't know how to deal with that… so I'm inclined just to leave out the first commit about repairing the tests at all… 🙃

Reply:
all commits are squashed upon merge anyways; in the master branch nobody will see how many commits you made or your commit messages

Reply:
as far as the other tests, run them without your changes. if they still fail without your changes, you don't need to worry about them. we'd only need to be concerned if your changes broke the tests, or if the urls aren't matched anymore

Reply:
OK. In that case, I don't see any errors for the test that I've added (`PBS_2`), and fixing the rest is still better than before.